sig-contributor-experience/triage-team/triage.md

```diff
@@ -0,0 +1,43 @@
+
+Speed up management of issues and PRs labeled with `sig/contributor-experience`.
+
+The SIG Contributor Experience [issues](https://github.com/kubernetes/community/labels/sig%2Fcontributor-experience) and [PRs](https://github.com/kubernetes/community/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+label%3Asig%2Fcontributor-experience+) are labeled with `sig/contributor-experience`.
+
+Following are few example searches on issue and PR:
+* [Open issues](https://github.com/kubernetes/community/labels/sig%2Fcontributor-experience)
+* [Open issues for a specific milestone](https://github.com/kubernetes/community/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3Asig%2Fcontributor-experience+milestone%3AMay)
+* [Open PRs](https://github.com/kubernetes/community/pulls?utf8=%E2%9C%93&q=is%3A+pr+is%3Aopen+label%3Asig%2Fcontributor-experience+) 
+* [Open PRs for a specific milestone](https://github.com/kubernetes/community/pulls?utf8=%E2%9C%93&q=is%3Apr+label%3Asig%2Fcontributor-experience+milestone%3AMay+is%3Aopen)
+
+## Scope
+
+Manage incoming issues and PRs that are already identified and labeled with the `sig/contributor-experience` particularly in the [Kubernetes community repository](https://github.com/kubernetes/community) but in general across various Kubernetes related GitHub repositories like [Kubernetes repository](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Asig%2Fcontributor-experience).
+
+## People involved
+
+Everyone is welcome to help manage issues and PRs but the work and responsibilities discussed in this document is created with [Triage Team](TODO-describe the team in a separate doc and add link) and [Triage Captain](TODO-describe the team in a separate doc and add link) role in mind.
+
+## Work involved
+
+The main work is to keep track of issues and PRs, ping people, add appropriate labels as needed and report the overall status to the SIG Leads.
+
+- **Validate if an issue is a bug**: Validate if the issue is indeed a bug by following these [instructions](https://github.com/kubernetes/community/blob/master/contributors/guide/issue-triage.md#validate-if-the-issue-is-bug).
+
+- **Determine it belongs to contributor-experience SIG**: An issue can be incorrectly labeled with the `sig/contributor-experience` label. If you think an issue does not belong to the `sig/contributor-experience`, add a comment stating so with a reason.
+
+- **Define priorities**: Define priority of an issue or PR by following these [instructions](https://github.com/kubernetes/community/blob/master/contributors/guide/issue-triage.md#define-priority).
+
+- **Verify important labels are in place**: Make sure that proper assignees are added in issue, reviewers are added to the PR and milestone is identified. If any of these labels are missing, you can add one if you are authorized to do so. If you can not assign labels or you can not decide the correct label, that’s fine, contact technical leaders of the SIG to do so.
```

Comment:
Wdyt about adding a separate point about adding `good-first-issue` and `help-wanted` labels if it makes sense? I think it could be helpful to encourage more contributors.

Reply:
Sure, that's good idea, we can add it as a separate point. I would, and I am sure we all, like to keep this doc small as possible. How about add a point with just this link? I will add a link to clarify the usage of the link  - https://github.com/kubernetes/community/blob/master/contributors/guide/issue-triage.md#help-wanted-issues

Reply:
I would recommend rewording this into a checklist style approach, and link out eg:
- has proper assignees (what is the definition of proper in this context?)
- has reviewers assigned
- has a milestone (is this the case for issues, PRs, or both? how do I decide what milestone?)

Also, what labels are you referring to here?